kore-rpc-types/src/Kore/JsonRpc/Types.hs

```diff
@@ -112,6 +117,7 @@ data ExecuteResult = ExecuteResult
     , state :: ExecuteState
     , nextStates :: Maybe [ExecuteState]
     , rule :: Maybe Text
+    , logs :: [LogEntry]
```

Comment:
We could maybe make this optional for better backwards-compatibility (found it because an integration test was missing the `logs: []` ). The field would be omitted if the list is empty